client/lib/signup/private-by-default.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+
+const debug = debugFactory( 'calypso:signup:private-by-default' );
+
+interface PrivateByDefaultSiteSettings {
+	flowName?: string;
+}
+
+/**
+ * Should the site be private by default
+ * @param siteSettings PrivateByDefaultSiteSettings
+ * @returns `true` for private by default & `false` for not
+ */
+export function shouldBePrivateByDefault( {
+	flowName = '',
+}: PrivateByDefaultSiteSettings ): boolean {
```

Comment:
This should make us confident enough that `siteSettings` won't be modified so there wouldn't be a need to shallow clone the object when calling this function.

```suggestion
}: Readonly<PrivateByDefaultSiteSettings> ): boolean {
```